src/ErrorHandler.js

```diff
@@ -1,21 +1,22 @@
-'use strict'
-import { silenceError, isHandled } from './inspect'
+import { silenceError } from './Promise' // deferred
```

Comment:
Oh @coveralls, why don't you like circular dependencies? Everything builds, tests and covers fine on my machine.